nexus-sdk-js/src/Organization/utils.ts

```diff
@@ -22,16 +27,34 @@ export async function createOrganization(
 }
 
 // TODO: refactor -> blocked by https://github.com/BlueBrain/nexus/issues/112
-export async function getOrganization(name: string): Promise<Organization> {
+/**
+ *
+ * @param orgLabel The organization label to fetch
+ * @param Object<revision, tag> The specific tag OR revision to fetch
+ */
+export async function getOrganization(
+  orgLabel: string,
+  options?: { revision?: number; tag?: string },
+): Promise<Organization> {
   try {
-    const orgResponse: OrgResponse = await httpGet(`/orgs/${name}`);
+    // check if we have options
+    let ops;
+    if (options) {
+      // it's rev or tag, not both. We take rev over tag
+      if (options.revision) {
+        ops = `?rev=${options.revision}`;
+      } else if (options.tag) {
+        ops = `?tag=${options.tag}`;
+      }
+    }
+    const orgResponse: OrgResponse = await httpGet(`/orgs/${orgLabel}${ops}`);
     // we want to know how many projects there are per organisation
     const listOrgsResponse: ListOrgsResponse = await httpGet('/projects');
     const projectNumber: number = listOrgsResponse._results
```

Comment:
Should be a separate function, it would be more semantic

Reply:
will fix in BlueBrain/nexus#339